.github/workflows/cypress_e2e.yml

```diff
@@ -48,5 +63,4 @@ jobs:
           NOTIFY_API_KEY: ""
           NOTIFY_MAGIC_LINK_TEMPLATE: ""
           SHORTENER_DOMAIN: "http://127.0.0.1:8000/"
-          SHORTENER_PATH_LENGTH: "6"
-          TABLE_NAME: "url_shortener_test"
+          SHORTENER_PATH_LENGTH: "6"
```

Comment:
This is more of a question I guess, but curious why this is set to 6 as opposed to the default 8.